index.html

```diff
@@ -66,12 +91,34 @@
         },
         editTodo: _.debounce(function(e) {
           this.model.setTitle(this.ui.todoTitle.val());
-        }, 500)
+        }, 1000)
       });
 
-      // The view for the entire application
-      var AppView = Backbone.View.extend({
-        el: $('#todoapp'),
+      // A view for an individual list item
+      var ListNavItem = Backbone.Marionette.ItemView.extend({
+        tagName:  "li",
+        className: "list-title",
+        template: _.template("<%= title %>"),
+        ui: {
+            'listTitle': '.list-title'
+        },
+        events: {
+          "click": "onClickListNavItem"
+        },
+        initialize: function(options) {
+          this.collection = options.collection;
+          this.listenTo(this.model, "change", this.render);
+        },
+        onClickListNavItem: function() {
+          // This needs to re-render the TodosContainerView
+          // Backbone.trigger('switch:list', this.model.)
+        }
+      });
+
+      // The view for the todos container
+      // ListsContainersView needs to refresh this view on render
+      var TodosContainerView = Backbone.View.extend({
```

Comment:
Same thing here with making it a `CollectionView` so that it will automatically handle add/remove events from the collection